CMakeLists.txt

```diff
@@ -334,7 +334,7 @@ add_library(jsoncpp OBJECT third_party/jsoncpp/jsoncpp.cpp)
 
 # General packaging configuration
 set(CPACK_GENERATOR "DEB")
-set(CPACK_DEBIAN_PACKAGE_MAINTAINER "anton.gerasimov@here.com")
+set(CPACK_DEBIAN_PACKAGE_MAINTAINER "patrick.vacek@here.com")
```

Comment:
lol

Reply:
maybe I should have asked you before indeed, sorry

Reply:
It's fine! I was just amused.